.gitignore

```diff
@@ -30,6 +30,11 @@ build/
 # spa_c_files builds
 pvlib/spa_c_files/build*
 
+# spa_c_files source
+pvlib/spa_c_files/spa.c
+pvlib/spa_c_files/spa.h
+pvlib/spa_c_files/spa_tester.c
+
```

Comment:
I think `pvlib/spa_c_files/spa_py.so` should be removed from the repo and also be added to the ignore file (also because the SPA tests are passing before SPA is installed!)